templates/frag/extractValue.hbs

```diff
@@ -0,0 +1,9 @@
+{{!--
+Extract the correct value from the JavaScript object for use with the function from fromJson.
+@param <this> CodegenProperty or CodegenSchema
+--}}
+{{#if (isFile schema)}}
+value
```

Comment:
I need to re-understand why we needed to do this. It may be something we can fix up in `api.hbs` when we populate `req.body`... but I need to see the generated code to be sure.

Reply:
We do this because the validation function for file parts calls `fileFromValue`, so we ensure we pass in the array containing the file object or the individual file objects (if the field supports a file array upload).